config.tfvars

```diff
@@ -41,10 +35,71 @@ instance_types = ["m5.2xlarge"]
 # Desired number of nodes that the node group should launch with initially.
 desired_capacity = 1
 
+################################################################################
+# Jira Settings
```

Comment:
Being picky here. Can these product specific config sections be ordered alphabetically.

Reply:
The reason behind this order is based on the number of users (bamboo has very low number of users so it goes to end). I think the most frequent used products go on top to be more accessible by the most of users. @errcode1202  WDYT?

Reply:
I didnt know that - Fair enough.